ble_controller/include/ble_controller_hci.h

```diff
@@ -16,7 +16,7 @@
  * The HCI packet format is described in the Bluetooth Core Specification,
  * i.e. in Core v5. Vol 2, Part E.
  * All APIs in this header file are expected to be called from the
- * same execution priority as @ref ble_controller_low_prio_tasks_process.
+ * same execution priority as mpsl_low_priority_process.
```

Comment:
So we cannot ref anymore?

Reply:
This kind of ref does not seem to work in the NCS documentation.